### ISSUE TITLE
Fuzz dll with Syzygy 

### ISSUE THREAD

**Issue by MostafaSoliman** [ivanfratric/winafl#115]
Hello Ivan,
Can i fuzz DLL with Syzygy, If yes, then will i need to instrument the DLL and add the __afl_persistent_loop() in my harness as shown in "test_static.cpp" ?

**Comment by ivanfratric** [comment]
Yes, that should work AFAIK, as long as you can instrument your .dll (it must be compiled with /PROFILE, see the readme)
